src/main/java/emu/grasscutter/command/commands/HealCommand.java

```diff
@@ -0,0 +1,36 @@
+package emu.grasscutter.command.commands;
+
+import emu.grasscutter.command.Command;
+import emu.grasscutter.command.CommandHandler;
+import emu.grasscutter.game.GenshinPlayer;
+import emu.grasscutter.game.props.FightProperty;
+import emu.grasscutter.server.packet.send.PacketAvatarFightPropUpdateNotify;
+import emu.grasscutter.server.packet.send.PacketAvatarLifeStateChangeNotify;
+import emu.grasscutter.server.packet.send.PacketEntityFightPropUpdateNotify;
+import emu.grasscutter.server.packet.send.PacketLifeStateChangeNotify;
+
+import java.util.List;
+
+@Command(label = "heal", usage = "heal|h",
+        description = "Heal all characters in your current team.", aliases = {"h"}, permission = "player.heal")
+public class HealCommand implements CommandHandler {
+    @Override
+    public void execute(GenshinPlayer sender, List<String> args) {
```

Comment:
`execute` is no longer valid, use `onCommand` instead

Reply:
> `onCommand`

Where is the `onCommand`?

Reply:
WTF, when did they change back🤔

Reply:
on `development` i changed it back to `execute` :p
execute is the proper term to use here